src/test/java/hw3/pages/HomePage.java

```diff
@@ -0,0 +1,134 @@
+package hw3.pages;
+
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.List;
+import java.util.Properties;
+
+import static org.testng.Assert.assertEquals;
+import static hw3.Const.BASE_URL;
+
+public class HomePage {
+    protected WebDriver driver;
+
+    @FindBy(className = "profile-photo")
+    private WebElement dropdownForm;
+
+    @FindBy(id = "name")
+    private WebElement loginForm;
+
+    @FindBy(css = "#password")
+    private WebElement passwordForm;
+
+    @FindBy(id = "login-button")
+    private WebElement enterButton;
+
+    @FindBy(xpath = "//*[@class='profile-photo']//span")
+    private WebElement user;
+
+    @FindBy(css = "ul.uui-navigation.nav")
+    private WebElement itemsOnTheHeader;
+
+    @FindBy(linkText = "HOME")
+    private WebElement homeMenuTab;
+
+    @FindBy(linkText = "CONTACT FORM")
+    private WebElement contactFormMenuTab;
+
+    @FindBy(linkText = "SERVICE")
+    private WebElement serviceMenuTab;
+
+    @FindBy(linkText = "METALS & COLORS")
+    private WebElement metalColorsMenuTab;
+
+    @FindBy(css = ".benefit-icon span")
+    private List<WebElement> images;
+
+    @FindBy(css = ".benefit-txt")
+    private List<WebElement> texts;
+
+    @FindBy(id = "frame")
+    private WebElement frame;
+
+    @FindBy(id = "button-frame")
+    private WebElement buttonFrame;
+
+    @FindBy(css = "ul.sidebar-menu")
+    private WebElement itemsOnTheLeftSection;
+
+
+    public HomePage(WebDriver driver) {
+        PageFactory.initElements(driver, this);
+        this.driver = driver;
+    }
+
+    public void openHomePage() {
+        driver.navigate().to(BASE_URL);
+    }
+
+    public void assertTitle(String titleExpected) {
+        assertEquals(driver.getTitle(), titleExpected);
+    }
+
+    public void login() throws IOException {
+        InputStream input = new FileInputStream("src/test/resources/user.properties");
```

Comment:
1) not closed input stream, use try-catch-with-resources
2) you will open the file + read properties every time you call this method - bad idea, too many resources. 
Create utils class for this purpose: read properties to the static field and use it in tests as const.